From ec8c3dc701cfd53e1ddc9817041e710fb4bbb0f3 Mon Sep 17 00:00:00 2001
From: Chuanhong Guo <gch981213@gmail.com>
Date: Sat, 9 Mar 2024 12:06:31 +0800
Subject: [PATCH] snfi: add support for GD5F1GQ5UE

This patch adds support for GigaDevice GD5F1GQ5UExxG to the
mtk-snfi driver in u-boot.

Signed-off-by: Chuanhong Guo <gch981213@gmail.com>
---
 drivers/mtd/mtk-snand/mtk-snand-ids.c | 4 ++++
 1 file changed, 4 insertions(+)

--- a/drivers/mtd/mtk-snand/mtk-snand-ids.c
+++ b/drivers/mtd/mtk-snand/mtk-snand-ids.c
@@ -115,6 +115,10 @@ static const struct snand_flash_info sna
 		   SNAND_MEMORG_1G_2K_64,
 		   &snand_cap_read_from_cache_quad_q2d,
 		   &snand_cap_program_load_x4),
+	SNAND_INFO("GD5F1GQ5UExxG", SNAND_ID(SNAND_ID_ADDR, 0xc8, 0x51),
+		   SNAND_MEMORG_1G_2K_128,
+		   &snand_cap_read_from_cache_quad,
+		   &snand_cap_program_load_x4),
 	SNAND_INFO("GD5F2GQ4UExIG", SNAND_ID(SNAND_ID_ADDR, 0xc8, 0xd2),
 		   SNAND_MEMORG_2G_2K_128,
 		   &snand_cap_read_from_cache_quad_q2d,
